func_tests/test_client.py

```diff
@@ -73,3 +75,70 @@ def test_batch_change(vinyldns_test_context):
     assert change2.record.ptrdname == 'change-test.vinyldns.'
     assert change2.type == RecordType.PTR
 
+    wait_until_recordset_deleted(vinyldns_test_context.client, r.changes[0].zone_id, r.changes[0].record_set_id)
```

Comment:
I am not getting a record set id back for some reason.  Here is a dump of the json I did get back...

```
{
   "userId": "ok",
   "userName": "ok",
   "comments": "comments",
   "createdTimestamp": "2019-08-07T02:08:09Z",
   "changes": [
      {
         "changeType": "Add",
         "inputName": "change-test.vinyldns.",
         "type": "A",
         "ttl": 200,
         "record": {
            "address": "192.0.2.111"
         },
         "status": "Pending",
         "recordName": "change-test",
         "zoneName": "vinyldns.",
         "zoneId": "45ff06bd-058a-4020-8c15-8acddec449e4",
         "validationErrors": [],
         "id": "71cf72b3-91f7-4aa1-8d05-aa4bedcfdad1"
      },
      {
         "changeType": "Add",
         "inputName": "192.0.2.111",
         "type": "PTR",
         "ttl": 200,
         "record": {
            "ptrdname": "change-test.vinyldns."
         },
         "status": "Pending",
         "recordName": "111",
         "zoneName": "2.0.192.in-addr.arpa.",
         "zoneId": "08652816-1eb6-4a7c-9ddf-3ec6a97d6b9f",
         "validationErrors": [],
         "id": "a1d3e736-985f-4362-98af-0a9b5f574ae9"
      }
   ],
   "status": "PendingProcessing",
   "approvalStatus": "AutoApproved",
   "id": "775670b4-586b-4187-ac11-3eab8e50fc00",
   "ownerGroupId": "5288ff9b-0a5d-4cb1-97e9-057c41eed481"
}
```

Reply:
Interestingly enough, there should be record set ids, and the status (if it is already complete) should be `Complete` me thinks